### PR TITLE
[3.7] docs(datetime pickers): Document AutoComplete parameter

### PR DESCRIPTION
**Pull request by dimodi**
plus some table polishing

### CODE REVIEW

File: components/dateinput/overview.md [hunk 1]
```diff
@@ -47,21 +47,21 @@ The Blazor Date Input generates events that you can handle and further customize
...
-|`Value`|`T` - expects a `DateTime` object|The value of the `DateInput`|
-|`TabIndex`|`int`|maps to the `tabindex` attribute of the HTML element. You can use it to customize the order in which the inputs in your form focus with the `Tab` key.|
-|`Placeholder`|`string`|maps to the `placeholder` attribute of the HTML element. The `Placeholder` will appear if the component is bound to nullable DateTime object - `DateTime?`, but will not be rendered if the component is bound to the default value of a non-nullable DateTime object. The Placeholder value will be displayed when the input is not focused. Once the user focuses it to start typing, the Format Placeholder (default or [customized one](#format-placeholder)) will override the Placeholder to indicate the format the date should be entered in|
-|`ValidateOn`|`ValidationEvent` enum <br/> `ValidationEvent.Input` | configures the event that will trigger validation (if validation is enabled). Read more at [Validation Modes for Simple Inputs]({%slug common-features/input-validation%}#validation-modes-for-simple-inputs)|
-
+|---|---|---|
+| `AutoComplete` | `string` | The `autocomplete` HTML attribute of the `input` |
```
Comment (ntacheva) [inline components/dateinput/overview.md:56]: Does it have a default value `"off"` as in the other components? If yes, we may add it here as well.

Comment (ntacheva) [inline components/dateinput/overview.md:56]:
```suggestion
| `AutoComplete` | `string` | The `autocomplete` HTML attribute of the `input`. |
```

File: components/datepicker/overview.md [hunk 2]
```diff
@@ -55,27 +55,28 @@ You can ensure that the component value is acceptable by using the built-in vali
...
 
 @[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
 | Attribute | Type and Default Value | Description |
-|----------|----------|----------|
-| `BottomView` | ` CalendarView` <br/> `CalendarView.Month` | Defines the bottommost view in the popup calendar to which the user can navigate to. |
-| `DebounceDelay` | `int` <br/> 150 | Time in milliseconds between the last typed symbol and the value update. Use it to balance between client-side performance and number of database queries.
-| `DisabledDates` | `List<DateTime>` | Specifies a list of dates that can not be selected. |
+|---|---|---|
+| `AutoComplete` | `string` <br /> (`"off"`) | The `autocomplete` HTML attribute of the `input` |
```
Comment (ntacheva) [inline components/datepicker/overview.md:66]:
```suggestion
| `AutoComplete` | `string` <br /> (`"off"`) | The `autocomplete` HTML attribute of the `input`. |
```

File: components/datetimepicker/overview.md [hunk 3]
```diff
@@ -54,22 +54,24 @@ When using the dropdown to edit dates, you must click the "Set" button to commit
...
-|`TabIndex`|`int?`|maps to the `tabindex` attribute of the HTML element. You can use it to customize the order in which the inputs in your form focus with the `Tab` key.|
-|`Placeholder`|`string`|maps to the `placeholder` attribute of the HTML element. The `Placeholder` will appear if the component is bound to nullable DateTime object - `DateTime?`, but will not be rendered if the component is bound to the default value of a non-nullable DateTime object. The Placeholder value will be displayed when the input is not focused. Once the user focuses it to start typing, the Format Placeholder (default or [customized one](#format-placeholder)) will override the Placeholder to indicate the format the date should be entered in|
-|`ValidateOn`|`ValidationEvent` enum <br/> `ValidationEvent.Input` |`ValidateOn` - configures the event that will trigger validation (if validation is enabled). Read more at [Validation Modes for Simple Inputs]({%slug common-features/input-validation%}#validation-modes-for-simple-inputs)|
+|---|---|---|
+| `AutoComplete` | `string` <br /> (`"off"`) | The `autocomplete` HTML attribute of the `input` |
```
Comment (ntacheva) [inline components/datetimepicker/overview.md:63]:
```suggestion
| `AutoComplete` | `string` <br /> (`"off"`) | The `autocomplete` HTML attribute of the `input`. |
```

File: components/timepicker/overview.md [hunk 4]
```diff
@@ -61,25 +61,25 @@ The time format specifiers in the `Format` control the tumblers available in the
...
-| `TabIndex` | `int?` | Maps to the `tabindex` attribute of the HTML element. You can use it to customize the order in which the inputs in your form focus with the `Tab` key.
-| `Placeholder` | `string` | Maps to the `placeholder` attribute of the HTML element. The `Placeholder` will appear if the component is bound to **nullable** DateTime object - `DateTime?`, but will not be rendered if the component is bound to the default value of a non-nullable DateTime object. The Placeholder value will be displayed when the input is not focused. Once the user focuses it to start typing, the Format Placeholder (default or [customized one](#format-placeholder)) will override the Placeholder to indicate the format the date should be entered in.
-| `ValidateOn` | `ValidationEvent` enum <br/> `ValidationEvent.Input` | Configures the event that will trigger validation (if validation is enabled). Read more at [Validation Modes for Simple Inputs]({%slug common-features/input-validation%}#validation-modes-for-simple-inputs).
-
+| `AutoComplete` | `string` <br /> (`"off"`) | The `autocomplete` HTML attribute of the `input` |
```
Comment (ntacheva) [inline components/timepicker/overview.md:72]:
```suggestion
| `AutoComplete` | `string` <br /> (`"off"`) | The `autocomplete` HTML attribute of the `input`. |
```

**Comment by ntacheva** [comment]
Related to https://github.com/telerik/blazor/issues/5298

File: components/dateinput/overview.md (hunk 1 above)
Comment (dimodi) [reply components/dateinput/overview.md:56]: No, it doesn't.

**Pull request merged**